core/labeled-replies-3.3.md

```diff
@@ -0,0 +1,36 @@
+---
+title: IRCv3.3 Labeled replies
+layout: spec
+copyrights:
+  -
+    name: "Alexey Sokolov"
+    period: "2015"
+    email: "alexey-irc@asokolov.org"
+---
+Name of this capability is `labeled-replies`.
+If client requests `labeled-replies`, it MUST also request `batch` capability.
```

Comment:
Might be a good idea to have the server NAK/DEL the cap if batch is not requested/goes away.